src/core/fbcontainer.cpp

```diff
@@ -36,67 +34,68 @@ EMGMResponse checkForActionEquivalentState(const CFunctionBlock &paFB, const EMG
   return EMGMResponse::InvalidState;
 }
 
-CFBContainer::CFBContainer(CStringDictionary::TStringId paContainerName, CFBContainer &paParent) :
-    mContainerName(paContainerName), mParent(paParent) {
-}
-
-CFBContainer::CFBContainer(CStringDictionary::TStringId paContainerName, CFBContainer &paParent, size_t paNumFBs) :
-    mContainerName(paContainerName), mParent(paParent) {
-  mFunctionBlocks.reserve(paNumFBs);
+CFBContainer::CFBContainer(CStringDictionary::TStringId paContInstanceName, CFBContainer &paParent) :
+    mContInstanceName(paContInstanceName), mParent(paParent) {
 }
 
 CFBContainer::~CFBContainer() {
-  for (TFunctionBlockList::iterator itRunner(mFunctionBlocks.begin()); itRunner != mFunctionBlocks.end(); ++itRunner) {
-    CTypeLib::deleteFB(*itRunner);
```

Comment:
Will this not attempt to delete internal FBs added from variables as well?

Reply:
You are totally right. I added code in CBasicFB to handle that case.